apps/src/applab/dropletConfig.js

```diff
@@ -1133,13 +1133,43 @@ export var blocks = [
 ];
 
 if (experiments.isEnabled(experiments.APPLAB_ML)) {
-  blocks.push({
-    func: 'getPrediction',
-    parent: api,
-    category: 'Data',
-    paletteParams: ['name', 'id', 'data', 'callback'],
-    params: ['"name"', '"id"', 'data', 'function (value) {\n \n}']
-  });
+  blocks.push(
+    {
+      func: 'getPrediction',
+      parent: api,
+      category: 'Data',
+      paletteParams: ['name', 'id', 'data', 'callback'],
+      params: ['"name"', '"id"', 'data', 'function (value) {\n \n}']
+    },
+    {
+      func: 'declareAssign_object',
+      block: `var object = {"key": "value"};`,
+      category: 'Variables',
+      noAutocomplete: true
+    },
+    {
+      func: 'declareAssign_object',
+      block: `var object = {"key": "value"};`,
+      category: 'Variables',
+      noAutocomplete: true
+    },
+    {
+      func: 'getValue',
+      parent: dontMarshalApi,
+      category: 'Variables',
+      paletteParams: ['object', '"key"'],
+      params: ['object', '"key"'],
```

Comment:
nit: Can you change `'object'` to an actual object literal like `{"key": "value"}`?
I think we've historically tried to make sure that everything in the toolbox actually works as soon as you drag it out